payers.md

```diff
@@ -8,3 +8,25 @@ the fixed yield at the end of the tenor, then the Receiver makes a profit, which
 implies a net loss for the Payer. A payer swap is therefore a bet on higher
 floating yields over a given tenor. Payers can speculate on overperforming
 market conditions compared to the guaranteed fixed interest rate of an asset.
+
+## Making a Profit
+
+Payers can make a profit under the following constraints and conditions.
+
+- The yield swap is a futures contract over a given tenor, e.g. 10 days.
+- The Payer pays the agreed upon fixed yield.
+- The Payer receives the floating yield actually realized after the tenor.
+- The fixed yield is **smaller** than the floating yield.
+
+![Payer Profit](image/payers/payer_profit.png)
+
+## Incurring a Loss
+
+Payers can incur a loss under the following constraints and conditions.
+
+- The yield swap is a futures contract over a given tenor, e.g. 10 days.
+- The Payer pays the agreed upon fixed yield.
+- The Payer receives the floating yield actually realized after the tenor.
+- The fixed yield is **greater** than the floating yield.
```

Comment:
Since the floating yield is the a priori "unknown", I would turn this around and say: "The floating yield is **smaller** than...